<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sqlEditor</class>
 <widget class="QDialog" name="sqlEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL Editor</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>104</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="executeButton">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>36</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Do SQL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>70</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>541</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="queryTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>401</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
